### PR TITLE
AC-981: multipart request generation

### PR DESCRIPTION
**Pull request by fredmaggiowski**
#### Checklist

- [x] your branch will not cause merge conflict with `master`
- [x] run `npm test`
- [x] tests are included
- [x] the documentation is updated or integrated accordingly with your changes
- [x] the code will follow the lint rules and style of the project
- [x] you are not committing extraneous files or sensible data

### CODE REVIEW

File: packages/api-explorer/src/components/Response/index.jsx [hunk 1]
```diff
@@ -57,20 +57,22 @@ class Response extends React.Component {
...
+              selected={responseTab}
+              onClick={this.setTab} 
+            >
+              <div  style={{maxHeight: '400px', overflow: 'hidden scroll'}}>
+                {responseTab === 'result' && <ResponseBody result={result} oauth={oauth} isOauth={!!securities.OAuth2} />}
+                {responseTab === 'metadata' && <ResponseMetadata result={result} />}
+              </div>
+            </BlockWithTab>
+          ) : (
+            <div style={placeholderStyle}>Try the API to see Results</div>
```
Comment (dibericky) [inline packages/api-explorer/src/components/Response/index.jsx]: FormattedMessage?

**Comment by dibericky** [comment]
LGTM

File: packages/api-explorer/src/lib/fetch-multipart.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+export const createMultipartBody = (formData) => {
+  const data = new FormData();
+  Object.keys(formData.formData).forEach((key) => {
+    const dataString = formData.formData[key];
+    if (dataString === undefined) {
+      return;
+    }
+    // Explode data string into component to be appended.
+    const actualData = dataString.split('base64,')[1];
```
Comment (davidebianchi) [inline packages/api-explorer/src/lib/fetch-multipart.js]: Comment with a formData example

**Comment by davidebianchi** [comment]
LGTM

**Pull request merged**